### ISSUE TITLE
bug: Two ways to change version in app info screen

### ISSUE THREAD

**Issue by oSumAtrIX** [ReVanced/revanced-manager#1994]
### Bug description

The correct way to "Undo" a choice is to "Go back" to the previous screen. Having a separate setting to change the version doesn't make sense in the sense of a progression:

![image](https://github.com/ReVanced/revanced-manager/assets/13122796/c7b2e62e-0b3c-4a4d-8839-aa27bcfa984c)

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The bug is only related to ReVanced Manager

**Comment by Ushie** [comment]
Perhaps this screen should be removed, it serves no purpose, instead the patch selector screen should take it's spot instead and from there we can arrive at the patching process

**Comment by oSumAtrIX** [comment]
Makes sense, although having an "Overview" before starting to patch is good. Mabe after the patch selector?

**Comment by Ushie** [comment]
Oh, I remember why we didn't want to throw the user into the patches screen

The user is expected to use the default selection of patches, making the patch selection page a forced step in the flow may cause the user to meddle when they don't really know what they're doing, those who want to change things will know how to, without adding an unnecessary step to the masses and making them think they have to do something

**Comment by oSumAtrIX** [comment]
A recommended Skip (Fab) button can be added

**Comment by Axelen123** [comment]
This issue will be dealt with by the downloader plugin branch
